crates/turborepo-scm/src/manual.rs

```diff
@@ -323,6 +325,14 @@ mod tests {
                 .unwrap();
         assert_eq!(hashes, expected);
 
+        // set a hash for an ignored file
+        for tuple in file_hash.iter_mut() {
```

Comment:
Non-blocking nit: You can destructure this tuple, like [here](https://github.com/vercel/turbo/pull/7105/files#diff-f8a7d7bb95a7cb29aa009a7bf275bcee7a3442065f307a55fc7e97c669d96d14R337). Then you don't need `.0` and `.2`

Reply:
Ah nice!